eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/AbstractLiteClient.java

```diff
@@ -46,4 +53,22 @@ public LiteClientConfig getLiteClientConfig() {
     public void shutdown() throws Exception {
         logger.info("AbstractLiteClient shutdown");
     }
+
+    public CloseableHttpClient setHttpClient() throws Exception {
```

Comment:
Declare that throwing a specific exception may be better than throw Exception.

Reply:
TrustManager[] tm = new TrustManager[]{new MyX509TrustManager()}; 

This line is throwing generic Exception. So it might not be appropriate to declare specific Exception in the method signature.

Reply:
since like you said the constructor `MyX509TrustManager()` throws a generic exception, we can wrap it into an `EventMeshException` object (`throw new EventMeshException(e);`)

this way we can just declare that `EventMeshException` would be thrown

Reply:
@ruanwenjun has a comment to in other part of my PR, mentioning not to wrap generic exception into EventMeshException. I am fine with either way, actually.

Reply:
I see

Reply:
In fact, this is my opinion, feel free to discuss.